sdk/src/test/tutorial/java/com/finbourne/lusid/tutorials/ibor/ReferencePortfolio.java

```diff
@@ -0,0 +1,149 @@
+package com.finbourne.lusid.tutorials.ibor;
+
+import com.finbourne.features.LusidFeature;
+import com.finbourne.lusid.ApiClient;
+import com.finbourne.lusid.ApiException;
+import com.finbourne.lusid.api.InstrumentsApi;
+import com.finbourne.lusid.api.PortfoliosApi;
+import com.finbourne.lusid.api.ReferencePortfolioApi;
+import com.finbourne.lusid.model.*;
+import com.finbourne.lusid.utilities.*;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.*;
+import java.util.stream.IntStream;
+
+import static org.junit.Assert.assertEquals;
+
+
+public class ReferencePortfolio {
+    private static List<String> instrumentIds;
+    private static PortfoliosApi portfoliosApi;
+    private static ReferencePortfolioApi referencePortfoliosApi;
+
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        ApiConfiguration apiConfiguration = new ApiConfigurationBuilder().build(CredentialsSource.credentialsFile);
+        ApiClient apiClient = new ApiClientBuilder().build(apiConfiguration);
+
+        portfoliosApi = new PortfoliosApi(apiClient);
+        referencePortfoliosApi = new ReferencePortfolioApi(apiClient);
+        InstrumentsApi instrumentsApi = new InstrumentsApi(apiClient);
+
+        InstrumentLoader instrumentLoader = new InstrumentLoader(instrumentsApi);
+        instrumentIds = instrumentLoader.loadInstruments();
+
+        Collections.sort(instrumentIds);
+    }
+
+    @Test
+    @LusidFeature("F39")
+    public void create_reference_portfolio() throws ApiException {
+
+        // Declare name, code of the portfolio to create
+        String F39PortfolioCode = "F39J_ReferencePortfolioCode";
+        String F39PortfolioName = "F39J_Reference Portfolio name";
```

Comment:
The feature number of this might change in the future, so I'd suggest removing "F39" references in the code, especially for the variable names - keep them simple as eg. portfolioCode, portfolioName.

Reply:
Also when renaming the portfolio code, make sure it's still relatively unique to avoid potential name clashes, as this will run in CI where there is a lot of data

Reply:
For uniqueness you may want to use the uuid pattern generally used in sdks. See Portfolios for an example.

Having the same name may also cause issues in future runs if the test didn't succeed in a specific run...see comment below around `deletePortfolio`